File: src/Python/Kujirahikouki_Rust/ch1_fizzbuzz.py
#PythonでFizzBuzz問題を解く
#1から100までを繰り返す
for i in range(1,101):
    #条件を1つずつ判定する
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print ("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)
        

File: src/Rust/kujirahikouki_Rust/ch1_fizzbuzz.rs
//RustでFizzBuzz問題を解く
fn main(){
    //1から100まで繰り返す
    for i in 1..101{
        //条件を1つずつ判定する
        if i % 3 == 0 && i % 5 == 0{
            println!("FizzBuzz");
        }else if i % 3 == 0{
            println!("Fizz");
        }else if i % 5 == 0{
            println!("Buzz");
        }else {
            println!("{}", i);
        }
    }
}